tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1229,3 +1229,54 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10272
+  Scenario: federated user downloads multiple files and folders as archive
+    Given using spaces DAV path
+    And using server "REMOTE"
+    And "Brian" has created the federation share invitation
+    And using server "LOCAL"
+    And "Alice" has accepted invitation
+    And user "Alice" has uploaded file with content "some data" to "textfile0.txt"
+    And user "Alice" has uploaded file with content "other data" to "textfile1.txt"
```

Comment:
can we upload images here?

Reply:
Yeah, but i think there will be problem while checking for the content of image in this step:
``` feature
And the downloaded zip archive should contain these files:
```

Reply:
So how have we tested for such cases before? Or there was no such scenario covered before?

Reply:
So far, there are no scenario for downloading images as zip.

Reply:
you should be able to check only the file/folder names. please check if that's true

Reply:
The following step checks if the file/folder exists in the zip or not.
``` feature
And the downloaded zip archive should contain these files:
```

if we want to just check the name of the folder then we can do this:
```
And the downloaded zip archive should contain these files:
      | name          | content    |
      | textfile0.txt | some data  |
      | textfile1.txt | other data |
      | my_data       |            |
      | more_data     |            |
```

Reply:
or maybe we should make another step :question:

Reply:
should be fine